rel/files/riak_cs.schema

```diff
@@ -0,0 +1,513 @@
+%% -*- erlang -*-
+
+%% Riak CS configuration
+
+%% == Basic Configuration ==
+
+%% @doc Riak CS http/https port and IP address to listen at for object
+%% storage activity
+{mapping, "listener", "riak_cs.listener", [
+  {default, {"{{cs_ip}}", {{cs_port}} }},
+  {datatype, ip},
+  {validators, ["valid_host"]}
+]}.
+
+{validator,
+ "valid_host",
+ "should be a valid host",
+ fun({Host, Port}) -> is_list(Host) andalso Port > 1024 andalso Port < 65536 end}.
+
+%% @doc Riak node to which Riak CS accesses
+{mapping, "riak_host", "riak_cs.riak_host", [
+  {default, {"{{riak_ip}}", {{riak_pb_port}} }},
+  {datatype, ip},
+  {validators, ["valid_host"]}
+]}.
+
+%% @doc Configuration for access to request
+%% serialization service
+{mapping, "stanchion_host", "riak_cs.stanchion_host", [
+  {default, {"{{stanchion_ip}}", {{stanchion_port}} }},
+  {datatype, ip},
+  {validators, ["valid_host"]}
+]}.
+
+{mapping, "stanchion.ssl", "riak_cs.stanchion_ssl", [
+  {default, {{stanchion_ssl}} },
+  {datatype, flag}
+]}.
+
+%% @doc Default cert location for https can be overridden
+%% with the ssl config variable, for example:
+{mapping, "ssl.certfile", "riak_cs.ssl.certfile", [
+  {datatype, file},
+  {commented, "$(platform_etc_dir)/cert.pem"}
+]}.
+
+%% @doc Default key location for https can be overridden with the ssl
+%% config variable, for example:
+{mapping, "ssl.keyfile", "riak_cs.ssl.keyfile", [
+  {datatype, file},
+  {commented, "$(platform_etc_dir)/key.pem"}
+]}.
+
+%% @doc Enable this to allow the creation of an admin user when
+%% setting up a system. It is recommended to only enable this
+%% temporarily unless your use-case specifically dictates letting
+%% anonymous users to create accounts.
+{mapping, "anonymous_user_creation", "riak_cs.anonymous_user_creation", [
+  {default, false},
+  {datatype, flag}
```

Comment:
isn't this should be `{flag, true, false}` ? [Flag description sounds like that](https://github.com/basho/cuttlefish/wiki/Datatypes#flag) but will test later.